dist/index.html

```diff
@@ -17,15 +18,15 @@
     <title>Leaderboard List App</title>
   <script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script></head>
 
-  <body>
```

Comment:
![image](https://github.com/grauJavier/leaderboard/assets/93039359/7a682f10-30ce-4b96-be68-904d4213ec37)

- [ ] It would be nice if you fix your live demo link because right now it is showing me a blank page with the leaderboard written on it maybe it is broken 😨 Kindly fix it or remove this link to make your readme more descriptive and professional 👍

Reply:
@HAMMAS-SALEEM I could do so, I wouldn't mind, but the project is still in `development` fase therefore not yet merged to `main`, once the PR gets approved it will and should be deployed. ¿That's the purpose of GitFlow and CR before deployment, or I got something wrong? Also, the deployment point you mention is not listed in the requirements of the project.

Reply:
Hey, @grauJavier I hope you are doing well dear 👍

Dear, the thing is that it was not a requirement to deploy the project but you have provided a link in your readme 🤔 so if you have provided a link in the readme then it is supposed to be working 😊 If you don't have plans to deploy your project then you can remove this link from your readme 👍 Once you would have a live link then you can add this LIve Demo section again in your readme file 👍

Reply:
@HAMMAS-SALEEM Thanks for your kind reply, again, the link provided is in `development` state using GitFlow for this purpose, therefore there is no need to have it working because is not open to the public. 
Also in your words: "**_it was not a requirement to deploy the project_**" Why did it get rejected making me lose time and forcing me to use another CR? Your kind advice could have been stated as [OPTIONAL].

Thanks anyway for your time. ✋

Reply:
Dear, the issue is that if your readme has any broken link or a link not working as expected then we are supposed to request change for that link. There are many other links which can be broken and not in the requirements which allows us to require changes such as a link of a tech stack. You are not required to add links of your tech stack but if any of that link is broken then it should be fixed to make your readme descriptive and professional

Reply:
Secondly, there is no force from my side to use another CR. If you think that the code review was not fair then you can ask for a second opinion from senior code reviewers by using the second opinion form which I have provided in my review

Reply:
Our intention is not to block you in your curriculum but our intention is that our students learn these things which shows their professionalism in their work. Microverse don't just make developers but it also make the developers a brand. So our primary focus to help them in not just technical part but also in the professional part as well

Reply:
@HAMMAS-SALEEM Thanks again for your time, vision, and experience. I sadly can't agree, as I stated my arguments before, and they seem to not resonate with you, there is not much I can add. Have a good week.

Reply:
@grauJavier I wish you the same dear, Have a great week :wave: